weldx/geometry.py

```diff
@@ -1469,14 +1649,25 @@ def __init__(
             raise ValueError("'radius' must have a positive value.")
         if angle <= 0:
             raise ValueError("'angle' must have a positive value.")
+
         self._radius = float(radius)
         self._angle = float(angle)
-        self._length = self._arc_length(self._radius, self._angle)
+
         if clockwise:
             self._sign_winding = -1
         else:
             self._sign_winding = 1
 
+        # todo change sign sign back to + and correct winding signs?
```

Comment:
Is this TODO still needed?

Reply:
depends... it basically just swaps the signs in the equation (so that it looks more like one would expect) and of the variable above. But in consequence, all tests have to be adjusted. Well, maybe I'll just try it and see how much work it is.